main.cc

```diff
@@ -13,13 +15,17 @@ int main(int argc, char *argv[]) {
 
 	if (argc != 2) {
 		cerr << "bad, command wrong" << endl;
-		throw 666;
+		throw "Please enter a valid command!";
 	}
 
 	// Step 1: Transform .pun to binary, and load the binary into memory
 	// Step 2: Have a program counter to point to instruction we take out
 		// Then, we decode it (into a specific instruction)
 		// Then we execute that instruction
 	Parser::writePUNInstructionsToMemory(argv[1], memory);
+
+	// for testing
+	parseInstruction("slime-massacre 0, 0, 100");
```

Comment:
ohhh we can encapsulate the entire main function in a try catch and print out the error we get from throwing instead! that probably works better instead of what we're doing right now...